### PR TITLE
Added stanford_s3fs_if to requirements.

### PR DESCRIPTION
**Pull request by sherakama**
Because we can now.

### CODE REVIEW

File: Environment/Anchorage/Install/S3FSFileSettings.php [hunk 1]
```diff
@@ -17,7 +17,6 @@ class S3FSFileSettings extends \AbstractInstallTask {
    *   Installation arguments.
    */
   public function execute(&$args = array()) {
-    module_enable(array('s3fs'));
```
Comment (jbickar) [inline Environment/Anchorage/Install/S3FSFileSettings.php:20]: Why was this here if it was also in `requirements()`?

Comment (sherakama) [inline Environment/Anchorage/Install/S3FSFileSettings.php:20]: Probably stemmed from copypasta and non-critical thinking.

**Comment by sherakama** [comment]
copypasta
